packages/gatsby-theme-docs/src/components/limit.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ContentNotifications } from '@commercetools-docs/ui-kit';
+import { useLimitByName } from '../hooks/use-limits';
+
+const Limit = (props) => {
+  const limit = useLimitByName(props.name);
+
+  if (!limit) {
+    const message = `${props.name} limit is not found.`;
+
+    if (process.env.NODE_ENV !== 'production') {
+      return <ContentNotifications.Error>{message}</ContentNotifications.Error>;
+    }
+
+    throw new Error(message);
+  }
+
+  return (
+    <>
+      {limit.number || ''}
+      {limit.text && limit.number ? <>&nbsp;</> : ''}
+      {limit.text || ''}
```

Comment:
```suggestion
      {[limit.number, limit.text].filter(Boolean).join()}
```

Reply:
> {[limit.number, limit.text].filter(Boolean).join('&nbsp;')}

Did you test that?

Reply:
Please - let's not make a code beauty contest out of this.  Thanks!